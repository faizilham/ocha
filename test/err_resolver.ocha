fn f () {}

a = 10; // not declared
f = 10; // can't assign value

if (true) {
    let b;
}

b = 10; // not declared
