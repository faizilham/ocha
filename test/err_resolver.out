not declared
can't assign value
not declared
